README.md

```diff
@@ -93,20 +93,44 @@ Copy your certificate files in `./certs` directory using following names:
 - guac-graphql-tls-certificate.pem"
 - guac-graphql-tls-certificate.key"
 
-6. Create Environment Variables with S3 and OIDC credentails
+6. Create Environment Variables for Storage, Events and OIDC 
 
 ```
 export TPA_STORAGE_ACCESS_KEY=<Storage Access Key>
 export TPA_STORAGE_SECRET_KEY=<Storage Secret Key>
-export TPA_STORAGE_OIDC_ISSUER_URL=<AWS Cognito or Keycloak Issuer URL. Incase of Keycloak endpoint auth/realms/chicken is needed>
-export TPA_STORAGE_OIDC_FRONTEND_ID=<OIDC Frontend Id>
+export TPA_OIDC_ISSUER_URL=<AWS Cognito or Keycloak Issuer URL. Incase of Keycloak endpoint auth/realms/chicken is needed>
+export TPA_OIDC_FRONTEND_ID=<OIDC Frontend Id>
 export TPA_OIDC_PROVIDER_CLIENT_ID=<OIDC Walker Id>
 export TPA_OIDC_PROVIDER_CLIENT_SECRET=<OIDC Walker Secret>
 export TPA_EVENT_ACCESS_KEY_ID=<Kafka Username or AWS SQS Access Key>
 export TPA_EVENT_SECRET_ACCESS_KEY=<Kafka User Password or AWS SQS Secret Key>
-export TPA_EVENT_BOOTSTRAP_SERVER=<Kafka Bootstrap Server URL, For AWS SQS it is not needed>
 ```
 
+In case of Kafka Events, create environmental variable for bootstrap server
```

Comment:
Add a number on this event section like the others ,to be visually separated as argument and not aggregated to the Envs section